apps/native/app/src/screens/settings/settings.tsx

```diff
@@ -230,6 +231,30 @@ export const SettingsScreen: NavigationFunctionComponent = ({
       })
   }
 
+  const updateLocale = (value: string) => {
+    client
```

Comment:
useHooks here? or maybe easer when updating cache

Reply:
Good point. The other update functions for notifications were like this so I just added this the same way. I think I can use `onCompleted` from the hook for updating the cache, not sure though if the implementation that is used now is faster and will not wait for the mutation to finish or not, will have to look into it. Will keep as is for now, might consider changing all the functions in a separate PR :)